docker/Dockerfile

```diff
@@ -54,59 +54,74 @@ RUN addgroup -S inventreegroup && adduser -S inventree -G inventreegroup
 WORKDIR ${INVENTREE_HOME}
 
 # Install required system packages
-RUN apk add --no-cache git make bash \
+RUN apk add --no-cache tini \
+    git make bash \
     gcc libgcc g++ libstdc++ \
     libjpeg-turbo libjpeg-turbo-dev jpeg jpeg-dev \
     libffi libffi-dev \
-    zlib zlib-dev
-
-# Cairo deps for WeasyPrint (these will be deprecated once WeasyPrint drops cairo requirement)
-RUN apk add --no-cache cairo cairo-dev pango pango-dev
-RUN apk add --no-cache fontconfig ttf-droid ttf-liberation ttf-dejavu ttf-opensans ttf-ubuntu-font-family font-croscore font-noto
-
-# Python
-RUN apk add --no-cache python3 python3-dev py3-pip
-
-# SQLite support
-RUN apk add --no-cache sqlite
-
-# PostgreSQL support
-RUN apk add --no-cache postgresql postgresql-contrib postgresql-dev libpq
-
-# MySQL support
-RUN apk add --no-cache mariadb-connector-c mariadb-dev mariadb-client
-
-# Install required python packages
-RUN pip install --no-cache-dir -U psycopg2 mysqlclient pgcli mariadb
-
+    zlib zlib-dev \
+    # Cairo deps for WeasyPrint (these will be deprecated once WeasyPrint drops cairo requirement) 
+    cairo cairo-dev pango pango-dev \
+    #fonts
+    fontconfig ttf-droid ttf-liberation ttf-dejavu ttf-opensans ttf-ubuntu-font-family font-croscore font-noto \
+    # Python
+    python3 python3-dev py3-pip \
+    # SQLite support
+    sqlite \
+    # Postgres support
+    postgresql postgresql-contrib postgresql-dev libpq \
+    # MySQL/Mariadb support
+    mariadb-connector-c mariadb-dev mariadb-client
+
+ENTRYPOINT ["/sbin/tini", "--"]
+
+ 
 FROM base as production
 # Clone source code
-RUN echo "Downloading InvenTree from ${INVENTREE_GIT_REPO}"
-
-RUN git clone --branch ${INVENTREE_GIT_BRANCH} --depth 1 ${INVENTREE_GIT_REPO} ${INVENTREE_HOME}
+RUN echo "Downloading InvenTree from ${INVENTREE_GIT_REPO}" \
+  &&  git clone --branch ${INVENTREE_GIT_BRANCH} --depth 1 ${INVENTREE_GIT_REPO} ${INVENTREE_HOME}
 
 # Checkout against a particular git tag
 RUN if [ -n "${INVENTREE_GIT_TAG}" ] ; then cd ${INVENTREE_HOME} && git fetch --all --tags && git checkout tags/${INVENTREE_GIT_TAG} -b v${INVENTREE_GIT_TAG}-branch ; fi
+RUN chown -R inventree:inventreegroup ${INVENTREE_HOME}/*
+
+WORKDIR ${INVENTREE_HOME}
+#temporary entrypoint copy test. Can be removed after merge
+RUN if test -f docker/init-server.sh; then \
```

Comment:
What's the purpose of this?

Reply:
Its to allow someone to test the compose files. The current prod repo that is checked out will not contain these files. (Unless my understanding of the workflow scripts is wrong [probably is]. 

That test can be removed if the build scripts use the correct branch when running the `production` image build.